src/QuoteBox.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './App.css';
+
+
+class QuoteBox extends Component {
+  handleClick(e) {
+    e.preventDefault();
+    this.forceUpdate();
```

Comment:
You shouldn't use `forceUpdate`
Keep `randomNum` in the state and init it in the constructor
```
constructor(props) {
  super(props)
  this.state = { randomNum: this._getRandomNum() }
}
_getRandomNum () {
   Math.floor(Math.random()*(this.props.quotes.length -1))
}
handleClick(e) {
  this.setState({ randomNum: this._getRandomNum()  })
}
```